src/components/vm/nics/nicBody.tsx

```diff
@@ -271,3 +304,363 @@ export const NetworkTypeAndSourceRow = ({
         </>
     );
 };
+
+export function portForwardText(pf: VMInterfacePortForward): string {
+    let text = "";
+
+    if (pf.address)
+        text += pf.address + ":";
+    if (pf.dev)
+        text += pf.dev + ":";
+    if (!pf.range.some(r => r.exclude != "yes"))
+        text += _("all");
+    for (let i = 0; i < pf.range.length; i++) {
+        const r = pf.range[i];
+        if (i > 0)
+            text += ",";
+        if (r.exclude == "yes")
+            text += _("excluding ");
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-3e464b33-20251031-114223-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)